opm/autodiff/FlowMainEbos.hpp

```diff
@@ -52,6 +53,28 @@
 
 namespace Opm
 {
+
+    /// \brief Gather cell data to global random access iterator
+    /// \tparam ConstIter The type of constant iterator.
+    /// \tparam Iter The type of the mutable iterator.
+    /// \param grid The distributed CpGrid where loadbalance has been run.
+    /// \param local The local container from which the data should be sent.
+    /// \param global The global container to gather to.
+    /// \warning The global container has to have the correct size!
+    template<class ConstIter, class Iter>
+    void gatherCellDataToGlobalIterator(const Dune::CpGrid& grid,
```

Comment:
it can stay in opm-simulators if you prefer. My point was rather that `FlowMainEbos.hpp` is the wrong file for a generic (i.e., non-`flow_ebos` related) function.